requirements.txt

```diff
@@ -315,3 +315,6 @@ caighdean==0.0.3 \
 # Required by caighdean
 nltk==3.2.5 \
     --hash=sha256:2661f9971d983db314bbebd51ba770811a362c6597fd0f303bb1d3beadcb4834
+
+django-webpack-loader==0.5.0 \
+    --hash=sha256:0a8536e36a30d719018cd4c5da6e9d2377771134e713c14e617bb484b4f0acce
```

Comment:
This should go up in the section for requirements with no dependencies. Also, please keep it alphabetically sorted.